Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/RCFCaster.java

```diff
@@ -40,7 +42,7 @@ public class RCFCaster extends ThresholdedRandomCutForest {
 
     public static boolean USE_INTERPOLATION_IN_DISTRIBUTION = true;
 
-    public static Calibration DEFAULT_CALIBRATION = Calibration.NONE;
+    public static Calibration DEFAULT_CALIBRATION = Calibration.SIMPLE;
```

Comment:
Will Calibration.MINIMAL causes the jitter issue?

Reply:
MINIMAL also resolved there jitter issues.